docs/adminguide/installation.rst

```diff
@@ -1,11 +1,11 @@
 .. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
-.. _doc_adminguide_install_client:
+.. _doc_adminguide_install_cli:
 
 
-Install Linux Parsec client
+Install Linux Parsec CLI
 ===========================
 
-Administrative operations must be performed with Parsec client for Linux. They can be performed from another machine.
+Administrative operations must be performed with Parsec CLI for Linux. They can be performed from another machine.
 
-See :ref:`Install Parsec (User Guide) <doc_userguide_install_parsec>`.
+See :ref:`Install Parsec CLI (User Guide) <doc_userguide_install_parsec>`.
```

Comment:
Move the instructions directly here to keep user guide install instructions as simple as possible (for the moment)